tests/config/database.php

```diff
@@ -5,10 +5,11 @@
     'connections' => [
 
         'mongodb' => [
-            'name'       => 'mongodb',
-            'driver'     => 'mongodb',
-            'host'       => '127.0.0.1',
-            'database'   => 'unittest',
+            'name'         => 'mongodb',
+            'driver'       => 'mongodb',
+            'host'         => '127.0.0.1',
+            'database'     => 'unittest',
+            'use_mongo_id' => true,
```

Comment:
we where already enabling `use_mongo_id` in `RelationsWithMongoIdTest`  `setUp` method ! why enforcing it here again? (if this tast runs after other tests then `tearDown` will be called later and every other test before that will be using mongo_ids enabled !